scripts/create-protection-market-curve3Crypto.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @notice Deploys the `CurveThreeTokens` trigger and creates a protection market
+ * @dev Ensure the desired TEST_CHAIN_FORK and RPC_URL env variables are set to target a specific chain
+ * @dev To deploy, you must define the CURVE_POOL environment variable, which is the address of the curve pool on the desired chain
+ */
+import hre from 'hardhat';
+import '@nomiclabs/hardhat-ethers';
+import { Contract, ContractFactory, utils } from 'ethers';
+import chalk from 'chalk';
+import { getChainId, getContractAddress, getGasPrice, logSuccess, logFailure, findLog, waitForInput, fundAccount } from '../utils/utils'; // prettier-ignore
```

Comment:
by default, let's leave `fundAccount` out since it's a helper and not for real deploy usage
```suggestion
import { getChainId, getContractAddress, getGasPrice, logSuccess, logFailure, findLog, waitForInput } from '../utils/utils'; // prettier-ignore
```

Reply:
good catch!